docs/installation.md

```diff
@@ -2,422 +2,439 @@
-  >
+#### 3.1.3 SQLite
 
+>
+> At this time there is no instruction for using SQLite on CentOS.
+>
 
-### *Installing dependencies on Debian and Ubuntu*
 
-Unless you've already done so, install Zonemaster *Engine* according to its own [installtion instruction](https://github.com/dotse/zonemaster-engine/blob/master/docs/installation.md).
+### 3.2 Installing dependencies on Debian and Ubuntu
 
-```sh
-sudo apt-get update
+```sh 
+sudo apt-get update 
 sudo apt-get install git libmodule-install-perl libconfig-inifiles-perl libdbd-sqlite3-perl starman libio-captureoutput-perl libproc-processtable-perl libstring-shellquote-perl librouter-simple-perl libclass-method-modifiers-perl libtext-microtemplate-perl libdaemon-control-perl 
-sudo cpanm -i Plack::Middleware::Debug Parallel::ForkManager JSON::RPC
+sudo cpanm -i Plack::Middleware::Debug Parallel::ForkManager JSON::RPC 
 ```
 
-Note: The Perl modules `Parallel::ForkManager` and `JSON::RPC` exist as Debian packages, but with versions too old to be useful for us.
-
-Install the chosen database engine and related dependencies, and start the database engine:
+Note: The Perl modules `Parallel::ForkManager` and `JSON::RPC` exist as Debian
+packages, but with versions too old to be useful for us.
 
-* MySQL
+Install the chosen database engine and related dependencies.
 
-  ```sh
-  sudo apt-get install mysql-server libdbd-mysql-perl
-  ```
+#### 3.2.1 MySQL
 
-* PostgreSQL
-
-  ```sh
-  sudo apt-get install libdbd-pg-perl postgresql
-  ```
+```sh 
+sudo apt-get install mysql-server libdbd-mysql-perl 
+```
 
-* SQLite
+#### 3.2.2 PostgreSQL
 
-  >
-  > At this time there is no instruction for using SQLite on Debian and Ubuntu.
-  >
+```sh 
+sudo apt-get install libdbd-pg-perl postgresql 
+```
 
+#### 3.2.3 SQLite
 
-### *Installing dependencies on FreeBSD*
+>
+> At this time there is no instruction for using SQLite on Debian and Ubuntu.
+>
 
-Unless you've already done so, install Zonemaster *Engine* according to its own [installtion instruction](https://github.com/dotse/zonemaster-engine/blob/master/docs/installation.md).
+### 3.3 Installing dependencies on FreeBSD
 
-```sh
-sudo pkg install p5-Config-IniFiles p5-DBI p5-File-Slurp p5-HTML-Parser p5-IO-CaptureOutput p5-JSON p5-JSON-RPC p5-Locale-libintl p5-libwww p5-Moose p5-Plack p5-Router-Simple p5-String-ShellQuote p5-Starman p5-File-ShareDir p5-Parallel-ForkManager p5-Daemon-Control p5-Module-Install p5-DBD-SQLite p5-Plack-Middleware-Debug
+```sh 
+sudo pkg install p5-Config-IniFiles p5-DBI p5-File-Slurp p5-HTML-Parser p5-IO-CaptureOutput p5-JSON p5-JSON-RPC p5-Locale-libintl p5-libwww p5-Moose p5-Plack p5-Router-Simple p5-String-ShellQuote p5-Starman p5-File-ShareDir p5-Parallel-ForkManager p5-Daemon-Control p5-Module-Install p5-DBD-SQLite p5-Plack-Middleware-Debug 
 ```
 
-Install the chosen database engine and related dependencies, and start the database engine:
+Install the chosen database engine and related dependencies.

```

Comment:
I think is a bit unclear when you read the line. Please update to:

Now install the chosen database engine and related dependencies, and start the database engine, by following the instructions in one of the following subsections.